packages/at_client/pubspec.yaml

```diff
@@ -40,10 +40,17 @@ dependencies:
   meta: ^1.8.0
   version: ^3.0.2
 
+dependency_overrides:
```

Comment:
If the changes in at_commons are published, please remove the dependency_overrides and updated the hosted version

Reply:
sure @sitaram-kalluri. Will do that as soon as at_commons is published